lxd/main_nsexec.go

```diff
@@ -681,6 +681,41 @@ void forkgetnet(char *buf, char *cur, ssize_t size) {
 	// The rest happens in Go
 }
 
+void proxydevstart(char *buf, char *cur, ssize_t size) {
```

Comment:
Our `C` convention is usually to use underscores so `proxy_dev_start()`.

Reply:
We were just copying the style used by other internal commands such as `forkgetnet()` and `forkremovefile()`, but it has been fixed.